org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CodeLensHandler.java

```diff
@@ -78,11 +78,25 @@ public CodeLens resolve(CodeLens lens, IProgressMonitor monitor) {
 			String command = null;
 			List<Location> locations = null;
```

Comment:
You could initialize locations to Collections.EMPTY_LIST. I think that would eliminate a few lines of code below.